std_msgs/Header header
nav_msgs/MapMetaData info
namo_navigation/Obstacle[] obstacles
int8[] data